ballerina/Module.md

```diff
@@ -216,19 +216,20 @@ sql:ParameterizedQuery sqlQuery =
 
 #### Creating Tables
 
-This sample creates a table with two columns. One column is of type `int` and the other is of type `varchar`.
+This sample creates a table with three columns. The first column is a primary key of type `int`,
+while the second column is of type `int`, and the other is of type `varchar`.
 The `CREATE` statement is executed via the `execute` remote function of the client.
 
 ```ballerina
-// Create the ‘Students’ table with the  ‘id’, 'name', and ‘age’ fields.
+// Create the ‘Students’ table with the ‘id’, ‘name‘, and ‘age’ fields.
 sql:ExecutionResult result = 
                 check dbClient->execute(`CREATE TABLE student (
                                            id NUMBER GENERATED ALWAYS AS IDENTITY,,
                                            age NUMBER, 
                                            name VARCHAR(255), 
                                            PRIMARY KEY (id)
                                          )`);
-//A value of the sql:ExecutionResult type is returned for 'result'. 
+// A value of the sql:ExecutionResult type is returned for 'result'. 
```

Comment:
```suggestion
// A value of the `sql:ExecutionResult` type is returned for  the `result`. 
```
Please check if backticks are used instead of single quotes in all occurrences.